src/python/ensembl/core/models.py

```diff
@@ -103,7 +103,8 @@ class AttribType(Base):
     name = Column(String(255), nullable=False, server_default=text("''"))
     description = Column(Text)
 
-    seq_region_attrib = relationship("SeqRegionAttrib", back_populates="attrib_type")
+    seq_region_attributes = relationship("SeqRegionAttrib", back_populates="attrib_type")
```

Comment:
This attribute name change is going to affect [at least another repository](https://github.com/Ensembl/ensembl-genomio/blob/48087e0c9f17cfedd93321d764130400b292baba/src/python/ensembl/io/genomio/seq_region_dumper.py#L31), so unless this change is required, I would suggest leaving it as it was. Furthermore, other Python repositories seem to give it the same previous name, e.g. [ensembl-api](https://github.com/Ensembl/ensembl-api/blob/710082a1d891f89e30322164572878a94c84ec67/src/python/ensembl/core/data/model/models.py#L978).

Reply:
Not even sure why I changed that. Changed back